### PR TITLE
Add overview page, regenerate image_ops output.

### PR DESCRIPTION
**Pull request by MarkDaoust**
- Add overview page to the docs (copied from /README.md)
- Minor doc fixes
    - Stray H1s that should be H2s. 
    - Add backticks for api-links
    - Typos
- Add output back to image_ops.ipynb 
   - From #578 : "displaying results in image_ops  but that's something we can submit follow-up PRs for."

For: #226

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<div align="center">
```
Comment (seanpmorgan) [inline docs/overview.md]: So this seems to be a mirror of our repo's README (without badge links). Will this generate a page on the website or what is the rationale for having this here?

File: docs/tutorials/image_ops.ipynb [hunk 2]
```diff
@@ -1,535 +1,665 @@
 {
```
Comment (seanpmorgan) [inline docs/tutorials/image_ops.ipynb]: <p>Hmmmm, not sure why the quiet flag didn't supress this but could we remove the downloading outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (MarkDaoust) [reply docs/tutorials/image_ops.ipynb]: Sure. Done.

File: docs/overview.md (hunk 1 above)
Comment (MarkDaoust) [reply docs/overview.md]: Yes, that's the intent here. 

We make the site from this directory. So this fork of your README is what's generating the overview/install page:

https://www.tensorflow.org/addons/overview

Comment (seanpmorgan) [reply docs/overview.md]: Got it. Thanks!

**Pull request merged**